json/de.json

```diff
@@ -7,7 +7,6 @@
     "Add": "Hinzufügen",
     "Added.": "Hinzugefügt.",
     "Administrator users can perform any action.": "Administratoren können jede Aktion durchführen.",
-    "Administrator": "Administator",
```

Comment:
as for me, you deleted unnecessarily ... you should leave them in their original form ... words that are in "exclusions" files, they only exclude information in the ToDo.md report